java-manta-client/src/main/java/com/joyent/manta/client/MantaObjectOutputStream.java

```diff
@@ -310,7 +310,10 @@ public synchronized void close() throws IOException {
         } catch (InterruptedException e) {
             // continue execution if interrupted
         } catch (ExecutionException e) {
-            throw new MantaIOException(e);
+            MantaIOException mioe = new MantaIOException(e);
```

Comment:
Without access to `HttpRequest`/`HttpResponse`, this won't actually have context like the request-id, right?